README.md

```diff
@@ -10,7 +10,7 @@
 
 | Readium | Android min SDK | Android compile SDK | Kotlin compiler (✻) | Gradle (✻) |
 |---------|-----------------|---------------------|---------------------|------------|
-| 3.0.0   | 21              | 34                  | 1.9.22              | 8.2.2      |
+| 3.0.0   | 21              | 34                  | 1.9.23              | 8.6.0      |
```

Comment:
```suggestion
| 3.0.0   | 21              | 34                  | 1.9.24              | 8.4.0      |
```

Reply:
It's bumped to 8.6.0: https://github.com/readium/kotlin-toolkit/blob/1f42e22018ffb0dca937954a9843024052ee458c/gradle/wrapper/gradle-wrapper.properties

Reply:
Not the plugin. Actually I'm not sure which one matters for apps. And the Kotlin version looks still wrong.

Reply:
Any reason why not 8.7?

Reply:
Ha you're right for the Kotlin compiler. For Gradle it is the Gradle version and not AGP, because this is the one people need to use when integrating the project as a submodule.

> Any reason why not 8.7?

I used what AGP Upgrade Assistant suggested, I'm not sure why 8.7 is not offered. Not yet supported officially maybe?